### PR TITLE
Global features for Rick and Marty API

### PR DESCRIPTION
**Pull request by riittagerhards**
Get started with building a Rick and Morty API ´. Add global CSS and an appElement with header and main in main.js.

localhost_3000_(Galaxy S5)

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1,8 +1,61 @@
...
+*::after,
+*::before {
+  box-sizing: border-box;
+}
+
+:root {
+  /* COLORS */
+  --color-header-text: #202329;
+  --color-main-text: #e0ffff;
+  --color-hover-text: orange;
```
Comment (leonardpaulus) [inline style.css:13]:
```suggestion
  --color-hover-text: #ff9800;
```

File: style.css [hunk 2]
```diff
@@ -1,8 +1,61 @@
...
+  --color-background-header: white;
+  --color-background-main: rgb(46, 43, 43);
```
Comment (leonardpaulus) [inline style.css:15]: Maybe stick to one unit for all variables and use #hex here. :)

**Review by leonardpaulus — CHANGES_REQUESTED** [review]
Looks good to me overall. Good work :)

**Review by manusmd — APPROVED** [review]
lgtm! ✨

File: main.js [hunk 3]
```diff
@@ -1,6 +1,46 @@
...
+  );
+
+  const footerElement = createElement(
+    "footer",
+    {
+      className: "footer",
+    },
+    [
+      createElement("p", {
+        textContent: "I am footer",
```
Comment (DennisTarik) [inline main.js:38]:
```suggestion
        textContent: "I am  a footer",
```

File: style.css [hunk 4]
```diff
@@ -1,8 +1,61 @@
...
+
+h1,
+h2,
+h3,
+h4,
+h5 {
+  font-family: var(--font-family-headline);
+}
+
+header {
```
Comment (lmachens) [inline style.css:40]: Be careful using element selector. You might use a `header` in other components too

**Pull request merged**